test/jdk/java/awt/print/PrinterJob/ImagePrinting/NullClipARGB.java

```diff
@@ -40,8 +42,12 @@ public static void main( String[] args ) {
             PrinterJob pj = PrinterJob.getPrinterJob();
             pj.setPrintable(new NullClipARGB());
             pj.print();
-            } catch (Exception ex) {
-             throw new RuntimeException(ex);
+        } catch (Exception ex) {
+            if (ex instanceof PrinterException) {
+                throw new SkippedException("Printer is required for this test.  TEST ABORTED");
```

Comment:
I think for this test, we need to check `PrintServiceLookup.lookupPrintServices(DocFlavor.SERVICE_FORMATTED.PRINTABLE, null);`, like other tests do it. Let's not assume that `PrinterException` is thrown here only when printer is not configured. Add the block right at the beginning of `main`?